src/hooks/useActiveAccount.tsx

```diff
@@ -61,10 +62,11 @@ export const ActiveAccountContextProvider = ({
    */
   accountIdToSelect?: string;
 }) => {
+  const { isLoggedIn } = useAuth();
   const accountsResult = useRestQuery(
     'GET /v1/accounts',
     {},
-    { select: (data) => data.accounts },
```

Comment:
Unrelated fix: This was attempting to fetch accounts before the app was fully logged in.